modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -101,9 +101,9 @@ $ oc delete pod --selector logging-infra=fluentd
 ----
 
 [id=cluster-logging-collector-log-forward-examples-syslog-parms]
-=== Syslog parameters:
+== Syslog parameters:
```

Comment:
@mburke5678 Can you check the changes in this file please?

Reply:
@bergerhoffer I think I used `===` to make the text smaller. Seems to large as `==`. Is there a better/proper way to do this? Maybe definition lists?
While you are in the file, could you male _Syslog parameters:_ and _Additional RFC5424 syslog parameters_ consistent? One has a colon and one doesn't. Probably remove the colon?

Reply:
@mburke5678 Sure thing, removed the colon.

And yeah, you need to go in order of the heading level, you shouldn't use a lower level just because you want the font to be smaller.

If I had to make a suggestion here, I would say that these sections don't really belong in the procedure module, and should probably be reference modules included after the procedure.

Reply:
> And yeah, you need to go in order of the heading level, you shouldn't use a lower level just because you want the font to be smaller.
Makes sense. Not sure why I formatted it this way. Thanks for fixing!